ome/omero-parade
# Use vite.js instead of webpack

Replace webpack with vite.js.

This also updates all dependencies to latest versions.

To test:
 - check that parade works as usual
 - Just try a basic filter -> table -> plot workflow
